content/resources/documentation/architecture/overview.md

```diff
@@ -1,74 +1,87 @@
+
+
+## Core Modules
+<figure>
+<v-img alt="The DigiWF architecture with attached custom components like frone end, microservices and integration 
+artifacts. You can see the four provided DigiWF modules core, tasklist, integration and co-creation." contain 
+max-width="960" 
+src="resources/documentation/architecture/digiwf_core_modules.
+png" 
+lazy-src="preview_blue_yellow.png" ></v-img>
+<figcaption>The DigiWF Modules + possibilities to integrate custom components</figcaption>
+</figure>
 
-Bla
+The picture above shows a possible "full blown" DigiWF architecture including self crafted artifacts. Everything in blue is provided by the DigiWF project, but as you know we're open for integrations from every direction. So you could create and integrate:
 
-<img src="/background_oqun5d_c_scale,w_700.png" width="100%" class="light-img" alt=""/>
+- your own front ends
+- your own integration artifacts to communicate with your on premise or cloud infrastructure
+- your own (micro) services
 
-## First header
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+You can use any technology you want. There're only two preconditions:
+
+- your frontend technology must have a GraphQL client
+- your backen (services, integration artifacts) must be able to communicate with one of [Spring Cloud Streams](https://spring.io/projects/spring-cloud-stream) supported Binder implementations.
 
 <v-alert color="yellow darken-1" border="left" elevation="2" colored-border icon="mdi-robot-confused">
-This is a hint field!
+If you want use an other EventBus infrastructure than Apache Kafka, you have to configure this in your DigiWF configurations.
 </v-alert>
 
-<v-alert color="red darken-1" border="left" elevation="2" colored-border icon="mdi-robot-angry">
-This is a warn field!
-</v-alert>
+Of course - if you'll use Spring Boot in your backend components and VueJs as front end technology, you can use all cross-section components (like Spring Boot Starters, NPM components, ...) we have created for our components.
 
-<v-alert color="grey lighten-1" border="left" elevation="2" colored-border icon="mdi-robot-happy">
-This is a neutral field!
-</v-alert>
+### DigiWF Core
+DigiWF Core is the heard of DigiWF. As you can see in the section above it consists of 5 services. It has a GraphQL API to interact with ist from front end applications and a very generic API, that communicates with various backend systems over an event bus.
```

Comment:
heart  ..................... to interact with its frontend apllications ....

Reply:
Done.